src/infrastructure/organisations/api.js

```diff
@@ -108,6 +108,8 @@ const getUserOrganisation = async (userId, organisationId, correlationId) => {
   return await callOrganisationsApi(uri, 'POST', payload, correlationId);
 };
 
+
```

Comment:
spaces not required

Reply:
These have been removed. Looks like they've been there a while.